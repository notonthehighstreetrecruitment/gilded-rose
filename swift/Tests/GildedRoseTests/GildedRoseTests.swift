import Foundation
import XCTest
import ApprovalTests_Swift

@testable import GildedRose

class GildedRoseTests: XCTestCase {

    func testGildedRose_updateQuality() throws {
        let items = [Item(name: "Aged Brie", sellIn: 2, quality: 0)]
        let app = GildedRose(items: items);
        app.updateQuality();
        
        try Approvals.verify(app.items[0].description)
    }
}
