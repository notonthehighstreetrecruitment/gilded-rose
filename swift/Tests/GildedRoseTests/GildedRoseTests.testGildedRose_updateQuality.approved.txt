Aged Brie, 1, 1
